### PR TITLE
What Cant Go Wrong.

### PR DESCRIPTION
**Pull request by SELFHELL**
## About The Pull Request

You know that whole skill for Flails **AND WHIPS**, turns out whips didn't even exist. This PR resolves that inconsistency.

## Why It's Good For The Game

Something something adding unimplemented content. On a side note, I spent an hour trying to FIND the icons for item intents

Themselves
icons

### CODE REVIEW

**Comment by TheManWithNoHands** [comment]
We be coming closer to castlevania each day.

**Comment by Eonoc** [comment]
Crafting recipe has an oversight. Your forgot to change the product to the new whip. As it is, it'll just produce a leather cloak.

/datum/crafting_recipe/roguetown/leather/whip
	name = "leather whip"
	**_result = /obj/item/clothing/cloak/raincloak/brown_**
	reqs = list(/obj/item/natural/hide = 2)

**Comment by Real-MAGNUM** [comment]
Fix this and I'll TM ^

File: code/modules/roguetown/roguecrafting/leather.dm [hunk 1]
```diff
@@ -97,6 +97,10 @@
 	result = /obj/item/clothing/cloak/raincloak/brown
 	reqs = list(/obj/item/natural/hide = 2)
 
+/datum/crafting_recipe/roguetown/leather/whip
+	name = "leather whip"
+	result = /obj/item/clothing/cloak/raincloak/brown
```
Comment (Real-MAGNUM) [inline code/modules/roguetown/roguecrafting/leather.dm:102]:
```suggestion
	result = /obj/item/rogueweapon/whip/antique
```

**Comment by Real-MAGNUM** [comment]
nvm I did it for you. Testmerging.

**Comment by SELFHELL** [comment]
oh thanks, forgor to fix that

**Comment by CVSthePharmacy** [comment]
arm breaking meta

**Pull request merged**